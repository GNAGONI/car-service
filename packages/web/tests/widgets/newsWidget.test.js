import React from 'react'
import renderer from 'react-test-renderer'
import NewsWidget from '../../components/widgets/newsWidget'

it('renders correctly', () => {
  const tree = renderer.create(<NewsWidget />).toJSON()
  expect(tree).toMatchSnapshot()
})
