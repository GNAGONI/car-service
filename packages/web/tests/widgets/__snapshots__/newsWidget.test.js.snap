// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="news-widget"
>
  <section
    className="widget widget-media"
  >
    <h2>
      <strong>
        Latest
      </strong>
       News
    </h2>
    <div
      className="post-holder"
    >
      <article
        className="post"
      >
        <div
          className="img-thumbnail"
        >
          <a
            href="#"
          >
            <img
              alt="Waste hours on the Rolls-Royce Cullinan configurator"
              height="70"
              src="https://i.ytimg.com/vi/JplDNiSMHaM/maxresdefault.jpg"
              width="100"
            />
          </a>
        </div>
        <div
          className="text-area"
        >
          <h3>
            <a
              href="#"
            >
              Waste hours on the Rolls-Royce Cullinan configurator
            </a>
          </h3>
          <time
            className="date"
          >
            12.02.03
          </time>
        </div>
      </article>
      <article
        className="post"
      >
        <div
          className="img-thumbnail"
        >
          <a
            href="#"
          >
            <img
              alt="Waste hours on the Rolls-Royce Cullinan configurator"
              height="70"
              src="https://i.ytimg.com/vi/JplDNiSMHaM/maxresdefault.jpg"
              width="100"
            />
          </a>
        </div>
        <div
          className="text-area"
        >
          <h3>
            <a
              href="#"
            >
              Waste hours on the Rolls-Royce Cullinan configurator
            </a>
          </h3>
          <time
            className="date"
          >
            15.02.03
          </time>
        </div>
      </article>
      <article
        className="post"
      >
        <div
          className="img-thumbnail"
        >
          <a
            href="#"
          >
            <img
              alt="Waste hours on the Rolls-Royce Cullinan configurator"
              height="70"
              src="https://i.ytimg.com/vi/JplDNiSMHaM/maxresdefault.jpg"
              width="100"
            />
          </a>
        </div>
        <div
          className="text-area"
        >
          <h3>
            <a
              href="#"
            >
              Waste hours on the Rolls-Royce Cullinan configurator
            </a>
          </h3>
          <time
            className="date"
          >
            16.02.03
          </time>
        </div>
      </article>
    </div>
    <button
      className="widget-btn d-block btn btn-primary"
      disabled={false}
      onClick={[Function]}
      succes={true}
      type="button"
    >
      View more
    </button>
  </section>
</div>
`;
